incubator/hnc/test/e2e/issues_test.go

```diff
@@ -269,11 +275,11 @@ var _ = Describe("Issues that require repairing HNC", func() {
 		MustRun("kubectl hns create", nsChild, "-n", nsParent)
 		MustRun("kubectl hns create", nsChild, "-n", nsParent2)
 		// Subnamespace child should be created and have parent as the 'subnamespaceOf' annoation value:
-		RunShouldContain("subnamespaceOf: "+nsParent, 1, "kubectl get ns", nsChild, "-o yaml")
+		RunShouldContain("subnamespaceOf: "+nsParent, defTimeout, "kubectl get ns", nsChild, "-o yaml")
 		// Creating a 'test-secret' in the subnamespace child
 		MustRun("kubectl create secret generic test-secret --from-literal=key=value -n", nsChild)
-		// subns (anchor) child in parent2 should have 'status: conflict' because it's a bad anchor:
-		RunShouldContain("status: conflict", 1, "kubectl get subns", nsChild, "-n", nsParent2, "-o yaml")
+		// subns (anchor) child in parent2 should have 'status: Conflict' because it's a bad anchor:
+		RunShouldContain("status: Conflict", defTimeout, "kubectl get subns", nsChild, "-n", nsParent2, "-o yaml")
```

Comment:
Similar to the `[s]` right? I will remember to update e2e test too when making future API conversion changes.

Reply:
We should eventually make it part of our presubmits